lib/modules/manager/pub/schema.ts

```diff
@@ -0,0 +1,14 @@
+import { z } from 'zod';
+import { Yaml } from '../../../util/schema-utils';
+
+export const PubspecLockSchema = z.object({
+  packages: z.record(z.any()),
```

Comment:
Let's omit this field completely if it isn't being used anywhere in Renovate

Reply:
```suggestion
```